### PR TITLE
feat: backward compatibility with tutor14

### PR DESCRIPTION
**Pull request by Ian2012**
## Description

This PR adds compatibility with tutor14 and tutor15

Fixes: #95

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @Ian2012!  Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


This is currently a draft pull request. When it is ready for our review and all tests are green, click "Ready for Review", or remove "WIP" from the title, as appropriate.

File: tutoraspects/plugin.py [hunk 1]
```diff
@@ -312,13 +313,26 @@
...
+    for service, template_path, priority in MY_INIT_TASKS:
+        full_path: str = pkg_resources.resource_filename(
+            "tutoraspects", os.path.join("templates", *template_path)
+        )
+        with open(full_path, encoding="utf-8") as init_task_file:
+            init_task: str = init_task_file.read()
+        hooks.Filters.COMMANDS_INIT.add_item(
+            (service, init_task)
+        )  # pylint: disable=no-member
+except AttributeError as e:
```
Comment (bmtcril) [inline tutoraspects/plugin.py:326]: Can you comment why we're doing these in both places?

File: tutoraspects/plugin.py [hunk 2]
```diff
@@ -479,8 +489,11 @@ def dbt(command: string) -> list[tuple[str, str]]:
 
 
 # Add the command function to CLI_DO_COMMANDS:
-hooks.Filters.CLI_DO_COMMANDS.add_item(load_xapi_test_data)
-hooks.Filters.CLI_DO_COMMANDS.add_item(dbt)
+try:
+    hooks.Filters.CLI_DO_COMMANDS.add_item(load_xapi_test_data)
+    hooks.Filters.CLI_DO_COMMANDS.add_item(dbt)
+except AttributeError:
```
[the remark this answers is not in the document]
Comment (Ian2012) [reply tutoraspects/plugin.py:501]: I've added support for tutor14 trough `tutor local|dev|k8s <command>`

File: tutoraspects/plugin.py [hunk 3]
```diff
@@ -478,11 +489,25 @@ def dbt(command: string) -> list[tuple[str, str]]:
...
+)
 # Add the command function to CLI_DO_COMMANDS:
-hooks.Filters.CLI_DO_COMMANDS.add_item(load_xapi_test_data)
-hooks.Filters.CLI_DO_COMMANDS.add_item(dbt)
 
 
+try:
+    CLI_DO_COMMANDS = hooks.Filters.CLI_DO_COMMANDS
+except AttributeError:
+    fmt.echo_alert("Upgrade to tutor 15 to use tutor local do commands.")
```
Comment (bmtcril) [inline tutoraspects/plugin.py:502]: I think you can get rid of this message

File: tutoraspects/plugin.py (hunk 1 above)
Comment (bmtcril) [reply tutoraspects/plugin.py:323]: I think we still need a comment here explaining why this try/except is needed

**Review by bmtcril — APPROVED** [review]
Couple of nits, but otherwise lgtm!

File: tutoraspects/plugin.py (hunk 1 above)
Comment (Ian2012) [reply tutoraspects/plugin.py:328]: Done

File: tutoraspects/plugin.py (hunk 3 above)
Comment (Ian2012) [reply tutoraspects/plugin.py]: Done

**Comment by openedx-webhooks** [comment]
@Ian2012 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/tutor-contrib-aspects&entry.867055334=https://github.com/openedx/tutor-contrib-aspects/pull/139&entry.1484655318=https://github.com/Ian2012&entry.752974735=2023-07-05+16:22&entry.1917517419=2023-07-06+14:59&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**